### PR TITLE
decouple steps and making parallel jobs

### PR DESCRIPTION
**Pull request by SamYuan1990**
decouple steps and making parallel jobs in azp

Signed-off-by: SamYuan1990 <yy19902439@126.com>

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -9,32 +9,49 @@ schedules:
...
-- script: cd interop; go test util_test.go pkuImport_test.go tjfocImport_test.go
-  displayName: test on loading artifacts 
-- script: DIR=$PWD;cd $(Pipeline.Workspace)/java-gm; gradle build; cp -f $(Pipeline.Workspace)/java-gm/*.pem $DIR/interop/testdata; cd $DIR/interop;go test tjjavaImport_test.go
-  displayName: interactWithJava
+  - job:
+    displayName: Java-GM
+    strategy:
+      matrix:
+        LTS:
+          goVersion: '1.13'
```
Comment (davidkhala) [inline azure-pipelines.yml:45]: java 应该也可以搞对应的版本matrix
如果在java-gm这个里面其实不需要对go的版本测那么细致了，只选一个较新的版本就行

**Comment by SamYuan1990** [comment]
@davidkhala , please take a look at. seems we need CCS-GM covert into pkcs#8.
cc: @guijunchen

File: azure-pipelines.yml [hunk 2]
```diff
@@ -9,32 +9,74 @@ schedules:
...
-- script: cd interop; go mod vendor
-- script: cd pku-gm; go mod vendor
-- script: cd interop; go test util_test.go ccsExport_test.go pkuExport_test.go tjfocExport_test.go
-  displayName: build artifacts
-- script: cd interop; go test util_test.go pkuImport_test.go tjfocImport_test.go
-  displayName: test on loading artifacts 
-- script: DIR=$PWD;cd $(Pipeline.Workspace)/java-gm; gradle build; cp -f $(Pipeline.Workspace)/java-gm/*.pem $DIR/interop/testdata; cd $DIR/interop;go test tjjavaImport_test.go
-  displayName: interactWithJava
+  - job:
+    displayName: Go-GM-PKU
```
Comment (davidkhala) [inline azure-pipelines.yml:43]: Why we need to make the first 2 jobs parallel?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -9,32 +9,74 @@ schedules:
...
+      displayName: Clone GMSSL
+    - script: cd $(Pipeline.Workspace)/GmSSL; SYSTEM=`uname -s` ./config --prefix=/usr/local/gmssl; make; sudo make install_sw  
+      displayName: Config, make and install GMSSL
+    - checkout: self
+    - task: GoTool@0
+      inputs:
+        version: $(goVersion)
+    - script: cd interop; go mod vendor
+    - script: cd pku-gm; go mod vendor
+    - script: cd interop; go test -v util_test.go ccsExport_test.go pkuExport_test.go tjfocExport_test.go
```
Comment (davidkhala) [inline azure-pipelines.yml:64]: Why pkuExport here? PKU lib is self-contained in its home

**Comment by davidkhala** [comment]
Stuck in CCS-gm pkcs8 format mis-alignment

File: interop/go.sum [hunk 4]
```diff
@@ -2,10 +2,18 @@ cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMT
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/Hyperledger-TWGC/ccs-gm v0.0.0-20200917142018-eeebcfc2577c h1:E9Fce5+w9aNdHzFAy+rFKwW+t0f4idMj3zJb/JXRsfk=
```
Comment (davidkhala) [inline interop/go.sum:3]: Can these be cleaned up? Will `go mod tidy` work?

Comment (SamYuan1990) [reply interop/go.sum:3]: will try

Comment (SamYuan1990) [reply interop/go.sum]: fixed.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,13 @@
 # fabric-gm-plugins
 
 - Home of interoperation tests among [pku-gm](https://github.com/Hyperledger-TWGC/pku-gm), [tjfoc-gm](https://github.com/Hyperledger-TWGC/tjfoc-gm) and [ccs-gm](https://github.com/Hyperledger-TWGC/ccs-gm)
-  
\ No newline at end of file
+  
+
+- Current version for each gm libs. At Dec-31-2020
```
Comment (davidkhala) [inline README.md:6]: 版本信息不需要详尽的写在README里，除非像tjfoc-gm这样的，是一个release版

**Review by davidkhala — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (SamYuan1990) [reply README.md:6]: 嗯，那就看看他们能不能出release版本了。

**Review by davidkhala — APPROVED** [review]
LGTM

**Pull request merged**